CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - BREAKING: Fix terraform route modification behavior when switching nat_gateway_mode
 
+### Added
+
+- Add support for Terraform `v0.15`
+
```

Comment:
added should go before fixed... we shoudl release 0.14. as a specific version or at least reserve a version number for it before 0.15